### PR TITLE
*: Remove coldef package

### PR DESCRIPTION
**Pull request by shenli**
@coocood @qiuyesuifeng

### CODE REVIEW

File: ast/misc.go [hunk 1]
```diff
@@ -473,3 +480,29 @@ func (n *GrantStmt) Accept(v Visitor) (Node, bool) {
...
+// Ident is the table identifier composed of schema name and table name.
+type Ident struct {
+	Schema model.CIStr
+	Name   model.CIStr
+}
+
+// Full returns an Ident which set schema to the current schema if it is empty.
+func (i Ident) Full(ctx context.Context) (full Ident) {
+	full.Name = i.Name
+	full.Schema = i.Schema
```
Comment (qiuyesuifeng) [inline ast/misc.go]: Seems no need to set `full.Schema` value here?

File: ddl/ddl.go [hunk 2]
```diff
@@ -468,16 +467,16 @@ func columnDefToCol(ctx context.Context, offset int, colDef *coldef.ColumnDef) (
...
-				if !evaluator.IsCurrentTimeExpr(v.Evalue) {
+			case ast.ColumnOptionOnUpdate:
+				if !evaluator.IsCurrentTimeExpr(v.Expr) {
 					return nil, nil, errors.Errorf("invalid ON UPDATE for - %s", col.Name)
 				}
 
 				col.Flag |= mysql.OnUpdateNowFlag
 				setOnUpdateNow = true
-			case coldef.ConstrFulltext:
+			case ast.ColumnOptionFulltext:
```
Comment (qiuyesuifeng) [inline ddl/ddl.go]: Comment alignment.

**Comment by shenli** [comment]
PTAL @qiuyesuifeng

**Comment by qiuyesuifeng** [comment]
LGTM

**Comment by coocood** [comment]
LGTM

**Pull request merged**